pkg/scheduler/core/generic_scheduler.go

```diff
@@ -741,18 +741,24 @@ func PrioritizeNodes(
 
 	workqueue.ParallelizeUntil(context.TODO(), 16, len(nodes), func(index int) {
 		nodeInfo := nodeNameToInfo[nodes[index].Name]
+		wg := sync.WaitGroup{}
```

Comment:
I don't think this change would improve the performance of the scheduling process since we have already calculated the score of each node concurrently and this may introduce unnecessary goroutine context switches.

Could you provide the benchmark numbers for this change as this is for performance improvement?

Reply:
You are right. Thank you for reviewing!